feat: [FX-2284] Add cancel callback for notifications (#2305)
toptal/picasso: .changeset/witty-beds-do.md

---
'@toptal/picasso': minor
---

> - When using `use-notification` hooks it started to call `onClose` callback when `Notification` is closing.

Reviewer: I'd make the description more concise.

```suggestion
Pass `onClose` to the notifications opened by `use-notification` hooks.
```

Author: @teimurjan done 👍
